src/locales/en_GB/messages.json

```diff
@@ -743,6 +743,19 @@
       }
     }
   },
+  "loggedInAsOn": {
```

Comment:
do not modify this file, please. only modify the `en` file for new languages. these other get added by our translation tools

Reply:
OK, done!